examples-next/ecommerce/mutations.ts

```diff
@@ -11,7 +12,7 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
   `,
   resolvers: {
     Mutation: {
-      checkout: async (root, { token }: { token: string }, context) => {
+      checkout: async (root, { token }: { token: string }, context: KeystoneContext) => {
```

Comment:
this should already be typed as KeystoneContext by `graphQLSchemaExtension`

Reply:
Yep, there's actually a few of these now I notice, going through and cleaning them up now.